skyportal/handlers/api/group.py

```diff
@@ -21,6 +21,23 @@
 _, cfg = load_env()
 
 
+def has_admin_access_for_group(user, group_id):
+    groupuser = (
+        GroupUser.query.filter(GroupUser.group_id == group_id)
+        .filter(GroupUser.user_id == user.id)
+        .first()
+    )
+    return (
+        len(
```

Comment:
No need to check length, set will evaluate to True unless it is empty

Reply:
Good suggestion -- implemented